ckanext/dgu/theme/templates/package/edit_form.html

```diff
@@ -603,10 +603,16 @@ <h3 style="font-weight: bold;">Primary Theme</h3>
     <hr/>
 
     <div class="well form-inline">
-      <h3>Mandate</h3>
+      <h3>Mandates</h3>
       <div class="form-group">
-        <input class="form-control" id="mandate" name="mandate" size="40" type="text" value="{{data.get('mandate', '')}}"/>
+        {%- set mandates = data.get('mandate', ['', '', '']) -%}
```

Comment:
I suggest you do always create a couple of blanks, whether or not there is a value: `data.get('mandate', []) + ['', '', '']`